akka-remote/src/main/scala/akka/remote/transport/netty/NettySSLSupport.scala

```diff
@@ -80,24 +76,18 @@ private[akka] class SSLSettings(config: Config) {
       case e: GeneralSecurityException ⇒ throw new RemoteTransportException("Server SSL connection could not be established because SSL context could not be constructed", e)
     }
 
-  def createSecureRandom(log: MarkerLoggingAdapter): SecureRandom = {
+  private[remote] def createSecureRandom(): SecureRandom = {
```

Comment:
Could we put some debug logs (back) in here, a before and after so that is easier to figure out that this is what is blocking actor system creation when waiting for '/dev/random'?

Reply:
No, actually, we cannot because we don't have access to a logger here.